tedana/decomposition/eigendecomp.py

```diff
@@ -318,13 +211,12 @@ def tedpca(catd, OCcatd, combmode, mask, t2s, t2sG,
 
         # Normalize each component's time series
         vTmixN = stats.zscore(comp_ts, axis=1).T
-        LGR.info('Making initial component selection guess from PCA results')
-        _, comptable, betasv, v_T = model.fitmodels_direct(
-                    catd, comp_ts.T, eimum, t2s, t2sG, tes, combmode, ref_img,
-                    mmixN=vTmixN, full_sel=False, label='mepca_',
-                    verbose=verbose)
-        # varex_norm overrides normalized varex computed by fitmodels_direct
-        comptable['normalized variance explained'] = varex_norm
+        comptable, _, _, _ = model.dependence_metrics(
+                    catd, OCcatd, comp_ts.T, eimum, t2s, tes, ref_img,
+                    reindex=False, mmixN=vTmixN, method=None,
+                    label='mepca_', out_dir=out_dir, verbose=verbose)
+        # varex_norm overrides normalized varex computed by dependence_metrics
+        comptable['real normalized variance explained'] = varex_norm
```

Comment:
Yeah, I reconsidered that wording a bit later. In #266 I change it to "original normalized variance explained." I can change that to something better though, if you want. I just want to retain both versions, for now, until we can validate `fitmodels_direct`.

I would argue, though, that the one estimated in `fitmodels_direct` _is_ kind of fake, since it doesn't match the one that comes directly from the PCA.

Reply:
Whoops, that should really go in #265. Pushing fixed name now.